examples/connectivity/plot_extract_regions_canica_maps.py

```diff
@@ -0,0 +1,85 @@
+"""
+Regions extraction using Canonical ICA maps
+===========================================
+
+This example shows how to extract each connected ICA map (mostly a 4D Nifti
+image/object) into a seperate brain activation regions and extracts timeseries
```

Comment:
A more correct formulation would be "This example shows how to use RegionExtractor to extract connected brain regions from whole brain ICA maps and use them to estimate a connectome."